### ISSUE TITLE
Newick trees without a root branch length fail to load

### ISSUE THREAD

**Issue by jameshadfield** [nextstrain/auspice.us#13]
**Current Behavior**  
Newick trees with incomplete branch lengths cannot be read by auspice.us. In particular, it is common to not define a branch length for the root node, such as:
```
((B:0.2,(C:0.3,D:0.4)E:0.5)F:0.1)A;
```

**Expected behavior**  
These trees are [valid newick](https://en.wikipedia.org/wiki/Newick_format), they should both load.

**How to reproduce**  

The following tree does not load 
```
((B:0.2,(C:0.3,D:0.4)E:0.5)F:0.1)A;
```

But this one does:
```
((B:0.2,(C:0.3,D:0.4)E:0.5)F:0.1)A:0;
```

**Your environment: if browsing Nextstrain online**  
 - auspice.us 0.8.0

**Additional context**  
Internal nodes without branch lengths (other than the root)  should still raise an error.

**Comment by abitrolly** [comment]
The parser https://github.com/nextstrain/auspice.us/blob/master/auspice_client_customisation/parseNewick.js

**Comment by babarlelephant** [comment]
@jameshadfield 

```
const addNodeName = (node) => {
	if (! ("node_attrs" in node)) node.node_attrs= {};
        if (! ("div" in node.node_attrs)) node.node_attrs.div= 0;
```

may solve the issue at least with getTreeStruct("(a,(b:2,c):3);")

**Comment by jameshadfield** [comment]
Closed by #27
